app/models.py

```diff
@@ -479,7 +479,11 @@ class UserEvent(Event):
 
     id = Column(types.Integer, ForeignKey('events.id'), primary_key=True)
     user_id = Column(types.Integer, ForeignKey('users.id'))
-    user = orm.relationship('User', backref='events')
+    # user = orm.relationship('User', backref='events')
+
+    user = orm.relationship('User',backref=backref('events',
+                         uselist=True,
+                         cascade='delete-orphan,all'))
```

Comment:
This looks good, but two things:

1. We don't need to keep the commented code.
2. Can you write some tests to prove that this works?